plugins/Web/plugin.py

```diff
@@ -146,12 +146,17 @@ class Web(callbacks.PluginRegexp):
     def noIgnore(self, irc, msg):
         return not self.registryValue('checkIgnored', msg.channel, irc.network)
 
-    def getTitle(self, irc, url, raiseErrors):
+    def getTitle(self, irc, url, raiseErrors, msg):
         size = conf.supybot.protocols.http.peekSize()
         timeout = self.registryValue('timeout')
+        headers = dict(utils.web.defaultHeaders)
+        language = conf.supybot.protocols.http.requestLanguage.getSpecific(
+                irc.network, msg.channel)()
+        if language:
+            headers['Accept-Language'] = language
```

Comment:
```
elif not language:
    del headers['Accept-Language']
```

Reply:
If we add that, then the global value will not be inherited, as it will just remove the inherited header from the dict altogether.

Reply:
The channel-specific value of `supybot.protocols.http.requestLanguage` inherits the global value already. So if it's empty, it means that either the global value is empty, or it was set to empty specifically for the channel.